test/pipeline/pipeline.import.azure.master.json

```diff
@@ -24,11 +24,308 @@
                     "branchFilters": [
```

Comment:
These are changes since last export.  If you'd prefer to see only my recent changes, let me know, I can check in current export from current pipeline and those will no longer show here.